src/main/java/org/torproject/onionoo/updater/RdnsLookupRequest.java

```diff
@@ -32,23 +35,64 @@ public RdnsLookupRequest(
   public void run() {
     this.lookupStartedMillis = System.currentTimeMillis();
     try {
-      String result = InetAddress.getByName(this.address).getHostName();
+      final List<String> verified_results = new ArrayList<>();
```

Comment:
Did this code pass `ant checks`? For local variables, avoid snake case. The only place where we use that is for old classes that are serialized using Jackson that we didn't update yet. Otherwise, avoid snake case.

Reply:
It did pass `ant checks`. I had made sure that everything passed before pushing.